src/main/resources/feedbackQuestionContribSubmissionFormTemplate.html

```diff
@@ -3,3 +3,20 @@
         ${contribSelectFragmentsHtml}
     </select>
 </div>
+<div>
+    <a href='#' data-toggle="modal" data-target="#more-info-equal-share-modal">${equalShareHelp}</a>
+</div>
+<div id='more-info-equal-share-modal' class="modal fade" role="dialog">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal">&times;</button>
+                <h4 class="modal-title">More info about the 'Equal Share' scale</h4>
+            </div>
+            <div class="modal-body">
+                <p>Equal share is a relative measure of individual contribution to a team.</p>
+                <p>For example, in a 3 person team, 'equal share' means a third of the work done.</p>
+            </div>
+        </div>
+    </div>
+</div>
```

Comment:
The newest linter will complain about this: add a line break at the end of the file.